api/HTMLAllCollection.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "5"
```

Comment:
This is later than the support for `document.all`. Was `document.all` something else before this?

Reply:
Looks like `document.all` just returned an `HTMLCollection` in older Safari versions:
<img width="422" alt="Screen Shot 2020-12-14 at 04 56 40" src="https://user-images.githubusercontent.com/5179191/102083650-c272f780-3dc8-11eb-9070-6bc41379cf18.png">

Reply:
It's tempting to suggest a `partial_implementation` for the range where `document.all` was a `HTMLCollection`, but unfortunately it seems that wouldn't capture what matters to web developers most.

Most of the things that work with an `HTMLAllCollection` also work with `HTMLAllCollection`. The ability to call `document.all` is only spec'd for `HTMLAllCollection`, so I tested http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8753 in Safari 4. That shows that their `HTMLCollection` was also possible to call as a function. So I think that `HTMLCollection` was later forked into `HTMLAllCollection` and in many small changes over time the two reached their current state.

Without careful testing, it'll be hard to write a note that says something that can be usefully interpreted by web developers. http://wpt.live/html/infrastructure/common-dom-interfaces/collections/htmlallcollection.html is pretty exhaustive, but it won't run on old Safari. Most of the tests were fixed between Safari 10.1 and 11.1, and so not related to `HTMLCollection` vs. `HTMLAllCollection`.

So... I don't have any suggestion for a note that would be useful.

Reply:
Hmm...  From what I can tell, both `HTMLCollection` and `HTMLAllCollection` have almost the same IDL, with a few slight differences.  I'm thinking that it may not be relevant enough to make a note, possibly?

Reply:
I agree, I don't think the differences are large enough to warrant a note.